containers/auditor-apel-plugin/Dockerfile

```diff
@@ -0,0 +1,57 @@
+#################
+# BUILDER IMAGE #
+#################
+
+FROM python:3.8.17-slim-bookworm AS builder
+
+# install build dependencies and create python env
+RUN set -ex                                                          \
+    && apt-get update                                                \
+    && apt-get -y install git                                        \
+    && python -m venv /env                                           \
+    && /env/bin/pip install --upgrade pip                            \
+    && apt-get -y autoremove && apt-get clean                        \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# activate python env
+ENV VIRTUAL_ENV /env
+ENV PATH /env/bin:$PATH
+
+# copy code
+COPY . /code
+# copy git dir (needed for setuptools_scm)
+COPY .git /code/.git
```

Comment:
Right, I was not sure if we can do this at this stage, but I guess since we don't have the publishing workflow in place, removing this should work. Will do this!